src/qt/bitcoingui.cpp

```diff
@@ -1034,37 +1035,37 @@ void BitcoinGUI::message(const QString &title, const QString &message, unsigned
     // Prefer supplied title over style based title
     if (!title.isEmpty()) {
         msgType = title;
-    }
-    else {
+    } else {
         switch (style) {
         case CClientUIInterface::MSG_ERROR:
-            msgType = tr("Error");
+            msgType = tr("ERROR");
```

Comment:
Why this is changing capitalization? All caps seems ugly.

Reply:
Fixed.